### ISSUE TITLE
Failed to reproduce Spikformer-4-384 on CIFAR10

### ISSUE THREAD

**Issue by TwillightRush** [ZK-Zhou/spikformer#33]
I tried to reproduce Spikformer-4-384 on CIFAR10 with the original code and the same hyperparameter in cifar10.yml, only got 91% top1 accuracy. 
Also, according to Issue #12("https://github.com/ZK-Zhou/spikformer/issues/12"), the author says they used 8 V100 GPUs. I'm using 1 A100 GPU, so I divided all parameters related to the learning rate (i.e.: lr, min_lr and warmup_lr) by 8 (according to the linear scaling rule), but my top1 accuracy was even worse: 86.37%.

I'm really confused, could anyone tell me why
MANY THANKS!
